apps/spruce/src/pages/waterfall/index.tsx

```diff
@@ -52,4 +49,20 @@ const PageContainer = styled.div`
   padding: ${size.m} ${size.l};
 `;
 
+/* Safari performance of the waterfall chokes if using overflow-y: scroll, so we need the page to scroll instead.
```

Comment:
This is such a wild quirk

Reply:
I can't tell by the below comment but is this override going to be permanent or is the plan to eventually update the Navbar component?

Reply:
Permanent, unless a Safari fix is introduced 🙃 pages with the LeafyGreen sidebar in particular need to utilize overflow-y